src/shared.js

```diff
@@ -804,6 +804,25 @@ export const updateSpecimen = async (array) => {
     return response.json();
```

Comment:
Same here if you have time. Thanks
```suggestion
    return await response.json();
```